mscore/menus.cpp

```diff
@@ -1435,6 +1435,33 @@ Palette* MuseScore::newTextPalette()
       stxt->setXmlText(tr("System Text"));
       sp->append(stxt, tr("System text"));
 
+      if (t != PaletteType::BASIC) {
+            StaffText* pz = new StaffText(SubStyleId::EXPRESSION, gscore);
```

Comment:
`SubStyleId` would need to get changed to `SubStyle` when cherry-picking for a 2.x branch